sdks/python/container/run_validatescontainer.sh

```diff
@@ -74,7 +75,7 @@ CONTAINER=us.gcr.io/$PROJECT/$USER/$IMAGE_NAME
 PREBUILD_SDK_CONTAINER_REGISTRY_PATH=us.gcr.io/$PROJECT/$USER/prebuild_python${PY_VERSION//.}_sdk
 echo "Using container $CONTAINER"
 
-if [[ "$ARCH" != "ARM" ]]; then
+if [[ "$ARCH" == "x86" ]]; then
```

Comment:
it would be more reliable to check for ARM in line 86 and error-out if the value is unrecognized.